app/geo/gui/src/ts/twxplore/gui/geo/attributeStrategies/classes/FrequencyFeatureAttribute.ts

```diff
@@ -0,0 +1,53 @@
+import {FeatureAttribute} from "./FeatureAttribute";
+import {MapFeatureAttributeState} from "../../states/map/MapFeatureAttributeState/MapFeatureAttributeState";
+import {setInitialFiltersNumeric} from "../functions/setInitialFilters/setInitialFiltersNumeric";
+import {FeatureAttributeName} from "../../states/map/FeatureAttributeName";
+import {TypeOfFeatureAttribute} from "../../states/map/TypeOfFeatureAttribute";
+import {KeplerFilterType} from "../../states/map/KeplerFilterType";
+import {KeplerFieldType} from "../../states/map/KeplerFieldType";
+import {Dispatch} from "redux";
+import {buildInitialFeatureAttributeStateNumeric} from "../functions/buildInitialFeatureAtttributeState/buildInitialFeatureAttributeStateNumeric";
+import {MapFeature} from "../../states/map/MapFeature";
+import {updateNumericAttributeStateOfFeatureType} from "../functions/updateAttributeStateOfFeatureType/updateNumericAttributeStateOfFeatureType";
+
+export class FrequencyFeatureAttribute implements FeatureAttribute {
+  updateAttributeStatesOfFeatureType(
```

Comment:
If this is all the delegation is, have an abstract parent class NumericFeatureAttribute?